onprem/onprem_instance.go

```diff
@@ -24,9 +23,7 @@ func (p *OnPrem) CreateInstance(ctx *lepton.Context) error {
 
 	imageName := c.CloudConfig.ImageName
 
-	if matched, _ := regexp.Match(`.img$`, []byte(imageName)); !matched {
-		c.CloudConfig.ImageName = imageName + ".img"
-	}
+	c.CloudConfig.ImageName = imageName
```

Comment:
This line can be removed altogether, since we have `imageName := c.CloudConfig.ImageName` above